src/hooks/usePlaceOrder.ts

```diff
@@ -81,7 +81,8 @@ export const usePlaceOrder = (): Result => {
             userAddress,
             buyTokenId,
             sellTokenId,
-            validUntil: validUntil || batchId + DEFAULT_ORDER_DURATION,
+            // validUntil time or max time
+            validUntil: validUntil ? batchId + validUntil : MAX_BATCH_ID, //batchId + DEFAULT_ORDER_DURATION,
```

Comment:
Can you update/remove the comment at the end of the line?

Reply:
yep